pybtp/btp.py

```diff
@@ -3353,6 +3353,13 @@ def gap_passkey_disp_ev_(gap, data, data_len):
 
     logging.debug("passkey = %r", passkey)
 
+    passkey = str(passkey)
+    # unpacking passkey to int loses leading 0s,
+    # let's add them back if lost
+    pk_len = len(passkey)
```

Comment:
https://docs.python.org/2/library/stdtypes.html#str.zfill

Reply:
now it uses `zfill()`